src/bp-templates/bp-nouveau/includes/profile-header-customizer-controls.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Customizer controls
+ *
+ * @since BuddyPress 3.0.0
```

Comment:
@shohel change the since tags and BuddyBoss

Reply:
@mehulkaklotar  Done